src/cms/templates/pages/page_xliff_confirm.html

```diff
@@ -29,21 +29,31 @@
     <div class="m-4">
       <form action="{% url 'confirm_xliff_import' region_slug=region.slug language_code=language.code%}" method="POST">
         <h2 class="heading pb-4">{% trans 'Review XLIFF Import' %}</h2>
-        <p class="pb-4">{% trans 'Please review the changes that will be done to the page translations below.' %}</p>
+        <p class="pb-4">{% trans 'Please review the changes that will be applied to the page translations below.' %}</p>
         <input type="submit" value="{% trans 'Confirm' %}" class="inline-block cursor-pointer bg-blue-500 hover:bg-blue-600 text-white h-full font-bold py-2 px-4 rounded p-4">
         <input type="button" value="{% trans 'Abort' %}" onclick="window.location='{% url 'pages' region_slug=region.slug language_code=language.code %}';" class="inline-block cursor-pointer bg-red-500 hover:bg-red-600 text-white h-full font-bold py-2 px-4 rounded p-4">
         <input type="hidden" value={{ upload_dir }} name="upload_dir">
         {% csrf_token %}
       </form>
     </div>
     {% for xliff_diff in translation_diffs %}
-    <div class="border border-green-500 m-4">
+    <div class="border border-{% if xliff_diff.error %}red{% else %}green{% endif %}-500 m-4">
+        {% if xliff_diff.error %}
+        <h2 class="heading font-normal m-4">{% trans 'Translation cannot be imported' %}</h2>
+        <p class="m-4">{% trans 'The page or target language does not exist.' %}
+        <ul class="list-disc m-4 pl-4">
+            <li class="">{% trans 'Title' %}: {{ xliff_diff.title }}</li>
+            <li class="">{% trans 'Page ID' %}: {{ xliff_diff.page_id }}</li>
```

Comment:
AFAICT this is the only place where we actually show the page ID in the user interface (beyond the address bar). However, I consider this a necessary information in this case, as the ID is used for mapping the XLIFF to a page and this is the only clue about what went wrong.